source/main.cpp

```diff
@@ -315,11 +315,17 @@ void GameLoop(PlayerInfo &player, const Conversation &conversation, const string
 				// The UI handled the event.
 			}
 			else if(event.type == SDL_KEYDOWN && !event.key.repeat
-					&& (Command(event.key.keysym.sym).Has(Command::FASTFORWARD)))
+					&& (Command(event.key.keysym.sym).Has(Command::FASTFORWARD))
+					&& !Command(SDLK_CAPSLOCK).Has(Command::FASTFORWARD))
 			{
 				isFastForward = !isFastForward;
 			}
 		}
+
+		// Special case: If fastforward is on capslock, update on mod state and not
+		// on keypress.
+		if(Command(SDLK_CAPSLOCK).Has(Command::FASTFORWARD))
+			isFastForward = SDL_GetModState() & KMOD_CAPS;
```

Comment:
Fun fact: this completely nullifies the fast forward interrupt setting, as the state of the caps lock will constantly force fast forward back on if the game is trying to interrupt it.

I don't imagine that will cause any issues, but just something to keep an eye out for.

Reply:
The override happens later, so I'm assuming that it will still interrupt?

Reply:
It will, but it will then be on as soon as the panel that was interrupting fast forward is popped off; the point of the fast forward interrupt was for the fast forward to be off after a panel is popped.